tests/core/vertex/_utils/test_calculate_cost.py

```diff
@@ -0,0 +1,9 @@
+"""Tests the `vertex._utils.calculate_cost` function."""
+
+from mirascope.core.vertex._utils._calculate_cost import calculate_cost
+
+
+def test_calculate_cost() -> None:
```

Comment:
this will need to be updated to test the actual cost calculation when updated

Reply:
updated